spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/repository/it/SpannerRepositoryIntegrationTests.java

```diff
@@ -181,6 +182,22 @@ public void queryMethodsTest() {
 				.findByTraderId("trader2");
 		assertThat(trader2TradesRetrieved).containsExactlyInAnyOrderElementsOf(trader2Trades);
 
+		assertThat(this.tradeRepository
```

Comment:
If you want, add test for limit in the method name and page request in the params. Which takes precedence?